client/src/workspace/layouts/WorkspaceBaseLayout.tsx

```diff
@@ -17,26 +46,39 @@ const WorkspaceRoot: React.FC = () => {
     );
 
   return (
-    <Layout
-      hasSider
-      style={{
-        backgroundColor: 'transparent',
-        gap: '20px',
-        paddingLeft: '20px',
-        paddingRight: '20px',
-        overflowX: 'auto',
-        overflowY: 'hidden',
-      }}
-    >
-      <Sider width={200} theme="light" breakpoint="md" collapsedWidth={0}>
-        <h1 className="headline headline-research" id="headline-data-depot">
-          <span className="hl hl-research">Tools and Applications</span>
-        </h1>
-        <JobStatusNav />
-        <AppsSideNav />
-      </Sider>
-      <Outlet />
-    </Layout>
+    <>
+      <div className={styles.breadcrumbWrapper}>
+        <AppsBreadcrumb
```

Comment:
If the breadcrumWrapper is only wrapping the breadcrumb, can we move this straight to the AppsBreadcrumb component, so we can just render `<AppsBreadcrumb ...>`?